Dockerfile

```diff
@@ -1,48 +1,23 @@
-FROM ubuntu:14.04
+FROM p4lang/behavioral-model:latest
```

Comment:
I don't know how Docker works, but I imagine it's dangerous to always depend on latest, since it may introduce breaking changes without warning.